### PR TITLE
Fix type error cannot read property async

### PR DESCRIPTION
**Pull request by AllenFang**
Try to fix https://github.com/revamp-with-react/ungit-react/pull/7#issuecomment-310895441

### CODE REVIEW

**Comment by AllenFang** [comment]
@CYBAI, please review, seems like Travis was pass now.

File: src-react/__tests__/middlewares/redux-api/ungit-config.test.js [hunk 1]
```diff
@@ -3,8 +3,9 @@ import sinon from 'sinon';
 import expect from 'expect.js';
 import configureStore from 'redux-mock-store';
 import { createStore, applyMiddleware } from 'redux';
-import { apiMiddleware, CALL_API } from 'redux-api-middleware';
+import { apiMiddleware } from 'redux-api-middleware';
 
+import { FETCH_HEADER } from '../../utils/constants';
```
Comment (CYBAI) [inline src-react/__tests__/middlewares/redux-api/ungit-config.test.js:8]: Can we require util without dots?
ex. `import { FETCH_HEADER } from 'utils/constants'`

**Review by CYBAI — CHANGES_REQUESTED** [review]
Thanks for your great work.
The fix looks good to me.
I think only the requiring path of utils in tests is the last part need updating.

File: src-react/__tests__/middlewares/redux-api/ungit-config.test.js (hunk 1 above)
Comment (AllenFang) [reply src-react/__tests__/middlewares/redux-api/ungit-config.test.js:8]: done

**Review by CYBAI — APPROVED** [review]

**Comment by AllenFang** [comment]
@CYBAI, I revert the lastest commit about resolving `'../../utils/constants'`, Because for fixing this issue, I add following in `package.json`:

```js
"moduleDirectories": [ 'node_modules', 'src-react/__test__' ]
```

But it most like cause a import naming conflict for `app`, for example, 

You can imagine this [line](https://github.com/revamp-with-react/ungit-react/blob/phase-1/src-react/__tests__/reducers/app.js#L4) and guess what module will be imported?!.

I try to avoid it bug but still fails, and also I think it's not very important that must handle this relative path in testing code, 

Please give some useful solution if you have, thanks

**Comment by CYBAI** [comment]
@AllenFang Okay, if I have better solution for this, I'll discuss with you.

**Pull request merged**